### PR TITLE
Docs/417579  proxy call to swagger

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/routes/assets.js [hunk 1]
```diff
@@ -1,55 +1,59 @@
...
-  method: 'GET',
-  path: '/static/assets/{path*}',
-  options: {
-    auth: false,
-    handler: {
-      directory: {
-        path: [
-          'node_modules/govuk-frontend/dist/govuk/assets'
-        ]
+module.exports = [
```
Comment (jbarnsley10) [inline app/routes/assets.js:3]: It looks like the only changes to this file is formatting, no actual code change?

File: app/routes/documentation.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+const { getEnvironmentVariable } = require('../lib/environment-helpers')
+const { admin } = require('../auth/permissions')
+
+module.exports = {
+  method: 'GET',
+  path: '/documentation',
+  options: {
+    auth: { scope: [admin] },
+    handler: async (_, h) => {
+      const allowedEnvironments = ['dev', 'snd', 'local']
```
Comment (jbarnsley10) [inline app/routes/documentation.js:11]: Can we put the allowedEnvironments in a constant file somewhere as you're using it a couple of times in separate route files

File: app/views/_swagger.njk [hunk 3]
```diff
@@ -0,0 +1,49 @@
+{% extends "govuk/template.njk" %}
+{% block head %}
+    <!--[if !IE 8]><!-->
+    
+    <link href="/static/css/core.08d92743fd36535a3b5a.css" rel="stylesheet">
```
Comment (jbarnsley10) [inline app/views/_swagger.njk:5]: I think these guids are re-generated each time a webpack build is done. You'll probably have to use the machnism like in _layout.template.njk to include these dynamic files

File: app/views/_swagger.njk [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            {
+            href: "/cookies",
+            text: "Cookies"
+            }
+        ]
+    }
+    })
+}}
+
+<script type="module" src="/static/js/core.08d92743fd36535a3b5a.js"></script>
```
Comment (jbarnsley10) [inline app/views/_swagger.njk:45]: Same as prev comment about synamic files

File: app/views/_swagger.njk [hunk 5]
```diff
@@ -0,0 +1,49 @@
+{% extends "govuk/template.njk" %}
```
Comment (jbarnsley10) [inline app/views/_swagger.njk:1]: I think this file should be excluded from the codebase as it gets generated from the template

**Review by jbarnsley10 — CHANGES_REQUESTED** [review]
Great stuff. Just a couple of very minor things

File: app/routes/assets.js (hunk 1 above)
Comment (whitewaterdesign) [reply app/routes/assets.js:3]: correct

**Review by jbarnsley10 — APPROVED** [review]
Nice

**Pull request merged**